platform/web/js/libs/library_godot_audio.js

```diff
@@ -430,12 +430,12 @@ const GodotAudioScript = {
 				// Read input
 				const inb = GodotRuntime.heapSub(HEAPF32, p_in_buf, p_in_size);
 				const input = event.inputBuffer;
+				const input_channels = input.numberOfChannels;
 				if (GodotAudio.input) {
-					const inlen = input.getChannelData(0).length;
-					for (let ch = 0; ch < 2; ch++) {
+					for (let ch = 0; ch < input_channels; ch++) {
 						const data = input.getChannelData(ch);
-						for (let s = 0; s < inlen; s++) {
-							inb[s * 2 + ch] = data[s];
+						for (let sample = 0; sample < data.length; sample++) {
+							inb[sample * input_channels + ch] = data[sample];
```

Comment:
Might be not a good idea to pass all of the channels of the input buffer, but thought that downmixing code should be kept in the same place for all `AudioDriver`s.